### ISSUE TITLE
datasets

### ISSUE THREAD

**Issue by LayYoona** [ivanhe123/Handwriting-Removal-DIS#1]
the dataset is gone, please provider a new link, thinks~

**Comment by ivanhe123** [comment]
No, it is not. Check the files tab on the dataset page:

![Image](https://github.com/user-attachments/assets/2bd1dd05-86a2-47d6-9f88-ecb3256d4236)

**Comment by LayYoona** [comment]
![Image](https://github.com/user-attachments/assets/e2c4b1e5-f597-4d8f-bdbc-51bc3a2c2653)

![Image](https://github.com/user-attachments/assets/2c9cd2bf-322f-4617-92fa-cd88349c4b60)

**Comment by LayYoona** [comment]
Is it due to private datasets

**Comment by LayYoona** [comment]
requirements.txt is not exist

**Comment by ivanhe123** [comment]
Now it should be fixed

**Comment by ivanhe123** [comment]
![Image](https://github.com/user-attachments/assets/ea14ec72-d475-4f2b-b32a-9113eab0469f)

**Comment by ivanhe123** [comment]
requirements txt added.

**Comment by LayYoona** [comment]
the dataset is ok

**Comment by LayYoona** [comment]
![Image](https://github.com/user-attachments/assets/f7327118-5744-43a4-a078-d638b2d67bc4)

**Comment by LayYoona** [comment]
![Image](https://github.com/user-attachments/assets/970fe7f3-9595-4db6-8e22-b73442a1d9ec)

**Comment by LayYoona** [comment]
Where do these Python libraries come from

**Comment by ivanhe123** [comment]
there is instructions on where to donwload the "libraries"

![Image](https://github.com/user-attachments/assets/c776e6f1-9b48-4658-959e-ff59c2f70fd3)

![Image](https://github.com/user-attachments/assets/c4ae573c-b00a-482c-90cd-77c2d1b06af8)

**Comment by LayYoona** [comment]
![Image](https://github.com/user-attachments/assets/e19c487d-ec52-4c05-a6d3-fa1261737e25) I followed the steps, but there is still an error here. Can you help me

**Comment by ivanhe123** [comment]
have you ran it yet?

**Comment by LayYoona** [comment]
![Image](https://github.com/user-attachments/assets/46eb53c7-7fe2-4855-ae08-f3ce28377250) I have resolved the above issue, how can I limit the running memory

**Comment by ivanhe123** [comment]
what file are you in?

**Comment by LayYoona** [comment]
run Inference.py

**Comment by LayYoona** [comment]
![Image](https://github.com/user-attachments/assets/0db2d3a4-c831-40fb-a5ee-0ce490652108) if i use cuda, it will fail

**Comment by LayYoona** [comment]
so i use cpu to run it

**Comment by ivanhe123** [comment]
yeah

is it fixed?

**Comment by LayYoona** [comment]
Yes, I solved this problem

**Comment by LayYoona** [comment]
I found that the erased image is very blurry, and there are some areas that differ from the clear printed image of the original image. Is this due to too little training data

**Comment by LayYoona** [comment]
![Image](https://github.com/user-attachments/assets/2c58d40f-c2da-4d3e-8030-ba1e8c2363fe) this is result

**Comment by LayYoona** [comment]
![Image](https://github.com/user-attachments/assets/cc4d6203-1953-4589-8956-6d71cd9e01fc)  this is origin

**Comment by IGoByALotOfNames** [comment]
yes. It IS due to too less training data. The original data provided only have about 800-1000 datas, which isn't sufficiently enough

**Comment by ivanhe123** [comment]
yes. It IS due to too less training data. The original data provided only have about 800-1000 datas, which isn't sufficiently enough

**Comment by LayYoona** [comment]
How can I obtain more mask data? If I have a regular image, how can I obtain the format data needed for training

**Comment by LayYoona** [comment]
![Image](https://github.com/user-attachments/assets/e3e8e51a-ec44-4afb-a2ae-9e2b2bb9cadd) I used a regular picture

**Comment by LayYoona** [comment]
![Image](https://github.com/user-attachments/assets/737d8aa2-1735-4728-bd97-93f0aaaafa7c) this is result......

**Comment by ivanhe123** [comment]
are you usign the model or it is a way for you to mask the image?
the model is trained for black and white images.

**Comment by LayYoona** [comment]
What I want to do is to eliminate handwritten content from pictures

**Comment by LayYoona** [comment]
If it is a training resource image, it can be very good, but using a regular image, it cannot effectively remove handwritten content

**Comment by LayYoona** [comment]
![Image](https://github.com/user-attachments/assets/550997b4-ccfd-4393-b1ff-5ef5ae8a61e1) train data

**Comment by LayYoona** [comment]
![Image](https://github.com/user-attachments/assets/8a310069-2b14-4c03-bfba-ae32ff0e6568) the result

**Comment by ivanhe123** [comment]
yes, because using a regular image, the lighting isn't very uniform. I suggest you to normalize the image pixles or convert the entire image to only black and white.

**Comment by LayYoona** [comment]
How to obtain a mask image from a regular image, just like training data

**Comment by ivanhe123** [comment]
Some phones have a scanner function build into it. If you are using Redmi, there is a "文档" function that allows you to scan a normal picture into black and white. It is very effective.

**Comment by fengyuncode** [comment]
The input_size in the inferrence.py file is not defined. How should this [input_size]be set

<img width="803" alt="Image" src="https://github.com/user-attachments/assets/46397fca-5ff4-4d1e-aa3c-103b42251251" />

**Comment by ivanhe123** [comment]
it should be 1024

**Comment by fengyuncode** [comment]
Using creat_mask.py requires pre annotated data, where should I obtain such data so that I can generate the mask dataset myself,thank

**Comment by ivanhe123** [comment]
creat_mask.py is basically conversion of the "https://github.com/AndSonder/HandWritingEraser-Pytorch" dataset from a segmentation format with labeled with background blue, printed characters green, and handwriting in red. Therefore, it is used to convert this format to background to be black and handwriting to be white, ignoring the printed characters.
